frontend-cert/js-projects/first-oop-lab/user-stories.md

```diff
@@ -0,0 +1,19 @@
+You should define a constant of type `Object` named `projectStatus` with the different project statuses you want to have such as `PENDING`, `SUCCESS`, or `FAILURE` with your preferred descriptions.
```

Comment:
Looking at the sheet, `Object` seems to only come up once in the lecture videos. 

So not too many campers will feel comfortable using the Object type.
They will probably feel more comfortable doing something like this

```js
const projectStatus = {
  PENDING: { description: "Pending Execution" },
  SUCCESS: { description: "Executed Successfully" },
  FAILURE: { description: "Execution Failed" },
};
```
while you solution is still valid, not to many campers are going to instinctively reach for the `Object` type and Symbol.

So we should update this user story to by more generalize to accept multiple correct answers

Reply:
I have switched from `Object` and `Symbol` to your proposed solution. I've also updated the wording of the stories a bit. Ready for another review.